src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -2,15 +2,32 @@
 
 import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
 
-import java.io.*;
+import java.io.BufferedReader;
```

Comment:
this change looks wrong. Can you please make sure to import the correct settings here to avoid such changes?